ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/customized-resource/cucushift-installer-rehearse-vsphere-ipi-customized-resource-workflow.yaml

```diff
@@ -0,0 +1,18 @@
+workflow:
+  as: cucushift-installer-rehearse-vsphere-ipi-customized-resource
+  steps: 
+    pre:
+    - chain: cucushift-installer-rehearse-vsphere-ipi-customized-resource-provision
+    post:
+    - chain: cucushift-installer-rehearse-vsphere-ipi-deprovision
+    env:
+      COMPUTE_NODE_CPU: "12"
+      COMPUTE_NODE_DISK_SIZE: "150"
+      COMPUTE_NODE_MEMORY: "24576"
+      COMPUTE_NODE_REPLICAS: "3"
```

Comment:
Does that mean the original setting - `2` not work?

Reply:
no. i just keep it has same default value with setting in [cucushift-installer-check-vsphere-customized-resource-ref.yaml](https://github.com/openshift/release/pull/50213/files#diff-f9d99f15b9ca1f8c95fef1677ff7beecdd1e1375745bbb146f517e0e600dabb9)

Reply:
Here is my suggestion, always keep consistent default values between `cucushift-installer-check-vsphere-customized-resource` and `ipi-conf-vsphere-customize-resource`, for `cucushift-installer-rehearse-vsphere-ipi-customized-resource-workflow.yaml`, we need to set the real customized value for testing, so suggest to change to compute node replicas to `2` to save some resource, also can meet our test coverage.

Reply:
ok.